### DISCUSSION TITLE
How can I pass elasticsearch host address

### DISCUSSION THREAD

**Discussion by ankitola** [jertel/elastalert2#286] [Q&A] [answered]
host = args.host if args.host else input('Enter Elasticsearch host: ')

I am getting this error while running elastalert on container.
Even I copied the config file on right place.
Can someone please help me to get out of this.

**Comment by ferozsalam** [comment, marked as answer]
Hi - I've converted this into a discussion as it fits better here.

You are also going to need to post more details to get useful help. 

- Have you followed the instructions [here](https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-docker-container)?
- Have you set the `es_host` value? 
- Have you attempted to start the container with a shell in order to check that your configuration file is being correctly mounted?

**Reply by ankitola** [reply]
Yes, I have set the value for **es_host**
Here is the my config file.
**rules_folder: example_rules
run_every:
  minutes: 1
buffer_time:
  minutes: 15
es_host: localhost
es_port: 9200**

**Reply by ankitola** [reply]
This is what I am getting even after mounting the config file inside the container. 
**podman run  -v /tmp/config.yaml:/opt/config/config.yaml --name elastalert elastalert
Enter Elasticsearch host: Traceback (most recent call last):
  File "/usr/local/bin/elastalert-create-index", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/elastalert/create_index.py", line 219, in main
    host = args.host if args.host else input('Enter Elasticsearch host: ')
EOFError: EOF when reading a line**

**Reply by jertel** [reply]
Your config mount point is incorrect. Please review the link @ferozsalam posted earlier.

**Reply by ankitola** [reply]
Oh, my bad.
Thanks now it is working fine.

**Reply by ankitola** [reply]
One more query if you guys can check here or should I create another issue with another heading.

My ealstalert alert was working fine for last 2 months but now I am not getting any alerts. 
But whenever I run elastalert with verbose mode, I am able to get an alert.
Do you think any silence issue with Elasticsearch?
